plugins/hub/osbuild.py

```diff
@@ -8,6 +9,7 @@
 sys.path.insert(0, "/usr/share/koji-hub/")
 import kojihub  # pylint: disable=import-error, wrong-import-position
 
+logger = logging.getLogger('koji.plugins.osbuild')
```

Comment:
Super nit, but probably we should do singular.
```suggestion
logger = logging.getLogger('koji.plugin.osbuild')
```

- https://pagure.io/koji/blob/master/f/plugins/hub/protonmsg.py#_25
- https://pagure.io/koji/blob/master/f/plugins/hub/tag2distrepo.py#_16

(it also changes the spacing, but that is ok)

Reply:
Yeah, I didn't notice, but there should be 2 newlines before and after the added line.

Reply:
Ah thanks for the hint with the lines. I didn't get that initially. Fixed that now.